proto/prehog/v1alpha/teleport.proto

```diff
@@ -1480,6 +1480,9 @@ message SubmitEventRequest {
   // PostHog timestamp
   google.protobuf.Timestamp timestamp = 2;
 
+  // teleport_version is the version of Teleport that submitted the event.
+  string teleport_version = 95;
```

Comment:
nit: maybe worthwhile to add a comment at the bottom mentioning 95 is taken.

Reply:
oh, btw the protobuf compiler will catch it if you accidentally reuse a field number